frontend/app.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+var sayHello = function (name) {
+    return "Hello, " + name + "!";
+};
+var user = 'Lana';
+alert(sayHello(user));
+//# sourceMappingURL=app.js.map
```

Comment:
the build results should not be under the git.
under the git, only the files in which we write the code .
files generated by the machine are not.